dev-env/layer2/config.yaml.tmpl

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: dev-env-layer2
+      protocol: layer2
+      addresses:
+      - SERVICE_V4_RANGE
```

Comment:
Is this missing SERVICE_V6_RANGE? It seems to be referenced below.

Reply:
I'm wondering IPv6 support in MetalLB layer2 mode. Is that supported?

Reply:
As far as I look kind document, default is IPv4, so IPv6 is not supported as default.

https://kind.sigs.k8s.io/docs/user/configuration/

If you want to enable, it should be taken care of additional issue/PR I guess.

Reply:
IPv6 is supported in layer2 mode.  It's fine if that's captured as a follow-up issue, though.  Addressing that is bigger than adding automated layer2 configuration, because we have to expose some configuration of the kind environment itself.